src/util.ts

```diff
@@ -304,6 +304,14 @@ export async function screenshot(client: Client): Promise<string> {
   return screenshot.data
 }
 
+export async function pdf(client: Client): Promise<string> {
+  const {Page} = client
+
+  const pdf = await Page.printToPDF()
```

Comment:
The options _do_ work at the CDP/Chrome level, so there may be an issue somewhere else. Perhaps the options aren't being passed along correctly—also the initial support for printing to pdf in headless-chrome was somewhat limited—are you trying in a recent version of Chrome (or via the Proxy?)

```js
Page.printToPDF({
  landscape: false,
  displayHeaderFooter: false,
  printBackground: true,
  scale: 1,
  paperWidth: 8.27, // aka A4
  paperHeight: 11.69, // aka A4
  marginTop: 0,
  marginBottom: 0,
  marginLeft: 0,
  marginRight: 0,
  pageRanges: '',
})
```